src/styles/catalog.scss

```diff
@@ -0,0 +1,102 @@
+.catalog {
+  padding-top: 50px;
+
+  &__container {
+    max-width: 944px;
+    margin: 0 auto;
+  }
+
+  &__row {
+    display: flex;
+    flex-wrap: wrap;
+  }
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  margin-bottom: 30px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__picture {
+    text-align: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 37px;
+  }
+
+  &__title {
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-color-dark;
```

Comment:
Add this code to a separate `card.scss` file. Because `catalog` and `card` - different BEM blocks.

Reply:
+